src/main/java/cat/udl/eps/softarch/mynftmkt/domain/NFT.java

```diff
@@ -29,6 +32,15 @@
 
     private String content;
 
+    @ManyToOne
+    @NotBlank
+    @JsonIdentityReference(alwaysAsId = true)
+    private User author;
+
+    @ManyToOne
+    @NotBlank
```

Comment:
The application is crashing because it can't validate if a relation is blank. I would remove these validations.